attributes/default.rb

```diff
@@ -20,9 +20,10 @@
 default['elasticsearch']['heap_mem_percent'] = 50
 default['elasticsearch']['node_master'] = false
 default['elasticsearch']['node_data'] = true
+default['elasticsearch']['node_ingest'] = false
 default['elasticsearch']['cluster_name'] = "elasticsearch"
 default['elasticsearch']['member_hosts'] = ["http://el.baritolog.com"]
-default['elasticsearch']['memory_lock'] = true
+default['elasticsearch']['memory_lock'] = false
```

Comment:
any reason why we set this to false?

Reply:
Previously, this set `false` as default then I set it to `true` for es clustering reason but forgot that the existing one isn't setup this attribute on their existing bootstrap_attribute. So, we set it as false and for the new app group we can setup itu on bootstrap_attribute.